ros_node:
  loop_rate: 100.0

# defining up to 4 robots
robots:

  robot_1:
    name: franka_1
    frame_id:
    motion_interface_index: 1
    topics:
      ee_pose: /robot1/franka_state_controller/franka_states
      desired_motions: /robot1/desired_motions
      motion_interface: /mocap_system/pose_arrays
    ref_T_B:
      rotation:
        - [1, 0, 0]
        - [0, 1, 0]
        - [0, 0, 1]
      translation: [0, 0, 0]


  robot_2:
    name: 
    frame_id:
    motion_interface_index: 2
    topics:
      ee_pose:
      desired_motions: 
      motion_interface:
    ref_T_B:
      rotation:
        - [1, 0, 0]
        - [0, 1, 0]
        - [0, 0, 1]
      translation: [0, 0, 0]


  robot_3:
    name:
    frame_id: 
    motion_interface_index: 3
    topics:
      ee_pose:
      desired_motions:
      motion_interface:
    ref_T_B:
      rotation:
        - [1, 0, 0]
        - [0, 1, 0]
        - [0, 0, 1]
      translation: [0, 0, 0]


  robot_4:
    name:
    frame_id:
    motion_interface_index: 4
    topics:
      ee_pose:
      desired_motions:
      motion_interface:
    ref_T_B:
      rotation:
        - [1, 0, 0]
        - [0, 1, 0]
        - [0, 0, 1]
      translation: [0, 0, 0]
